defaults/main.yml

```diff
@@ -10,3 +10,4 @@ journald_sync_interval: 0
 journald_forward_to_syslog: false
 journald_rate_limit_interval_sec: 30
 journald_rate_limit_burst: 10000
+journald_system_keep_free: 10
```

Comment:
```suggestion
```
I don't think we want to have a default here, for these reasons:

* This will make the role change the value unexpectedly for users who upgrade to this version of the role
* The default value seems reasonable: https://www.freedesktop.org/software/systemd/man/latest/journald.conf.html#SystemMaxUse=